src/test/suite/fixtures/recorded/actions/drinkArg.yml

```diff
@@ -0,0 +1,27 @@
+languageId: typescript
+command:
+  spokenForm: drink arg
+  version: 2
+  action: editNewLineBefore
+  targets:
+    - type: primitive
+      modifiers:
+        - {type: containingScope, scopeType: argumentOrParameter}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |
+    function helloWorld(foo: string, bar: number, baz: string) {}
+  selections:
+    - anchor: {line: 0, character: 40}
+      active: {line: 0, character: 40}
+  marks: {}
+finalState:
+  documentContents: |
+    function helloWorld(foo: string, , bar: number, baz: string) {}
+  selections:
+    - anchor: {line: 0, character: 33}
+      active: {line: 0, character: 33}
+  thatMark:
+    - anchor: {line: 0, character: 35}
+      active: {line: 0, character: 46}
```

Comment:
hmm a bit on the fence about the "that" mark here.  I'd be tempted to have it be the newly created arg, ie empty selection before the `, `, and in the future it would have the new `, ` in its removal range.  I think that would be consistent with how things work with eg `"bring to after"`: `"that"` refers to the newly created thing